blog/Arrival on Goteborg.md

```diff
@@ -0,0 +1,53 @@
+# Arrival on Goteborg 
+

```

Comment:
Maybe add a few words about basic introduction and backgroud in general?

Reply:
It can be after how many hours travelling from the city in China, transfter from somewhere and arrived Gothenburg, cost of flight ticket, temparature in Sweden, first impression of the country and the city, etc.